### PR TITLE
Exclusive Input + Documentation Changes

### LINKED ISSUE #487: Space Drag Toggle Control Binding
Being able to toggle space drag, similar to how we can toggle gravity would be great for switching games without switching our control bindings. Switching from games that are space drag friendly, such as VRChat, to games that aren't, such as Boneworks takes a minute.

### LINKED ISSUE #511: Add Steam Mention in Installer section.
perhaps add several re-directs to steam in Documentation just for ease.

one being in installer section.

### PR DESCRIPTION
**Pull request by ykeara**
# New Feature: Exclusive Input.

### New Action
- Exclusive Input Toggle - Toggles Exclusive Input active/in-active state

### Overview
When Enabled Via new Check Box in Settings Exclusive Input will block the Application Input or the OVRAS input Effectively giving you a "fn" key. This will allow controllers with limited buttons to more freely use key-binds.

### Technical Details
- By Design everything in the `System` and `Haptic (hidden)` action sets will always stay active
- By Design the Active/in-active toggle remembers through session, but you will always start in the same state which should be `application mode` 
- You Toggle between the `application mode` and the `OVRAS mode` via the new Action `Toggle Exclusive Input`
- Both the Action and the CheckBox are **OFF** by default
- By Design, Input is only Blocked to things that have actions assigned. i.e. if you have a and b bound (even if empty) they won't work w/ application , but if you don't have an OVRAS action bound (or empty action) to trigger it should still work normally
  - **Note** you should always leave one trigger un-bound as navigating dashboard will stop working while in `OVRAS mode`
- By Design our priority is active when we are in `OVRAS` mode, our priority is set to 0x01000001 - which is where OVR toolkit, and Desktop Portal are set to, This should allow co-priority though un-tested.

### Documentation Updates

- Added References To Steam and links to Windows/Linux Usage, This is not full branding yet, but a first step.
- Fixed a link to Kung's 3.0 guide
- Added Un-install Directions for Windows to Read-me.

### Testing

- Tested Briefly in VR via Valve Index in Windows no issues - ykeara

fixes #511 
(at least for now)
fixes #487

I think Travis Build Failure was just weird one off issue.

### CODE REVIEW

File: src/overlaycontroller.h [hunk 1]
```diff
@@ -176,6 +184,7 @@ class OverlayController : public QObject
 
     void Shutdown();
     Q_INVOKABLE void exitApp();
+    // Q_INVOKABLE void toggleExclusiveInput();
```
Comment (username223) [inline src/overlaycontroller.h:187]: Is this not supposed to be used?

Comment (ykeara) [reply src/overlaycontroller.h:187]: No, used a macro had that for testing forgot to clean it up Think I also might need to clean up a mutex that is not used as well.

Comment (username223) [reply src/overlaycontroller.h:187]: Cool, let me know on disc when you've cleaned it up and I'll take a look again.

**Review by username223 — APPROVED** [review]
Not tested in VR.

**Comment by username223** [comment]
All CI report back positive, you can just merge it.

**Pull request merged**